interface/src/app/analyze/analysis/analysis.js

```diff
@@ -95,7 +98,11 @@ AnnotationType, SampleBin) {
       }, {
         'name': 'samples',
         'type': 'ordinal',
-        'domain': {'data': 'activity_normalized', 'field': 'sample'},
+        'domain': {
+          'data': 'activity_normalized',
+          'field': 'sample_order._id',
```

Comment:
What does this sample_order._id field give us? Is it an index-like value telling us where each sample is supposed to go in the ordered set?